### DISCUSSION TITLE
Add option for disabling UI scaling in gameplay

### DISCUSSION THREAD

**Discussion by Kaillr** [ppy/osu#27420] [Ideas] [answered]
I believe there should be a setting where you can disable UI scaling in gameplay since it breaks half the skins I use. I want to use a UI scaling of 0.8x-0.9x but I cannot be bothered since a ton of skins break and cause many to look bad overall.

So if this would be an option in the future, it would really be appreciated!

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/26244

**Reply by Kaillr** [reply]
Must have overlooked that, my bad!
